db/consistency_level.cc

```diff
@@ -260,7 +260,7 @@ filter_for_query(consistency_level cl,
     size_t bf = block_for(erm, cl);
 
     if (read_repair == read_repair_decision::DC_LOCAL) {
```

Comment:
Nit for the commit message: I would put "Avoid an extra call to block_for in the fastpath." in the subject and leave the body empty. This way `git shortlog` would show something meaningful.

Reply:
ok